designer/client/components/Flyout/Flyout.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { useContext, useEffect, useLayoutEffect, useState } from "react";
 import FocusTrap from "focus-trap-react";
 import { FlyoutContext } from "../../context";
+import { DataContext } from "../../context";
 import { i18n } from "../../i18n";
 
 import "./Flyout.scss";
+import { bool } from "aws-sdk/clients/signer";
 
-export function useFlyoutEffect(props = {}) {
+export function useFlyoutEffect(props: {}) {
```

Comment:
why was this changed? these mean different things. `= {}` will default props to an empty  object `{}` if undefined is passed in. `: {}` is a typing.

I suppose it will be safe to remove if no components are passing in `props.show` anymore?